rsync_ssh.py

```diff
@@ -22,8 +22,10 @@ def console_print(host, prefix, output):
     print(output)
 
 
-def console_show(window=sublime.active_window()):
+def console_show(window=None):
     """Show console panel"""
+    if window is None:
+        sublime.active_window()
```

Comment:
BTW. The proper fix would be to change the argument to `window` instead of `window=None` and just return from this function if `window` is `None`.

Reply:
Ouch, lol, yeah. Too little coffee today. If that was a ST regression, then I'm happy to forget about this. This was a bit of a panic because my workflow completely broke this morning ... and yeah, it seems like this is in 4171 ...

Reply:
I think a proper fix would still be good because it looks like the original code could in some cases trigger a crash (in theory), if it's possible for window to be `None`. Or maybe the issue in those new builds is that `run_command` is not available to be used during plugin import so previously it was a noop.

Reply:
I'm happy to fix that fix if you like. Using a default argument factory like this isn't quite Pythonic as it has lots of edge cases ...